train.py

```diff
@@ -42,7 +44,10 @@ def get_arguments():
                         help='Whether to store advanced debugging information '
                         '(execution time, memory consumption) for use with '
                         'TensorBoard.')
-    parser.add_argument('--logdir', type=str, default=LOGDIR,
+    parser.add_argument('--logdir', type=str, default=get_default_logdir(),
+                        help='Directory in which to store the logging '
+                        'information for TensorBoard.')
+    parser.add_argument('--checkpoint_dir', type=str, default=LOGDIR,
                         help='Directory in which to store the logging '
```

Comment:
That's a good idea 👍
How about adding a `--restore_from` option that specifies which model to start from?
(And starts a new subdirectory in the root logdir)